docs/source/development-guide/schema-deployment/deploy-test-process.rst

```diff
@@ -0,0 +1,15 @@
+Deployment Test Process
+============
+
+Since Augur is a data centric system, software testing is not sufficient for ensuring the integrity of current and future implementations. We also test data collection against real repository lists for each release. 
+
+1. New developers will work on forks in their own repositories, generally. For for active contributors, we may create branches for everyone. 
+
+2. Commit and push your work no less frequently than the end of each day. 
```

Comment:
I don't think this should be a requirement. Developers need to commit when they are finished with their task, not at the end of the day; we should never be knowingly pushing commits that break the build. I propose rewording this to something like:
"Commit and push each unit of work as soon as it's completed. For example, if you are adding some tests, adding some metric documentation, and fixing a bug in a worker, as soon as you are finished one of those tasks, you need to commit and push that work to the correct branch. Then, start on the next task from a clean working tree. (If a task needs more than one commit, that is okay too).